tests/tekton-resources/pipelines/eks/awscli-cl2-load-with-addons-slos.yaml

```diff
@@ -97,9 +97,9 @@ spec:
     description: "default metric period"
     default: "300"
   - name: timeout-pia-pod-creation
-    default: "80s"
+    default: "40s"
```

Comment:
40 seconds for how many pods ?

Reply:
5000 pods with 200 scheduler QPS, I want to leave some space first